src/components/notes/editor-view.jsx

```diff
@@ -73,37 +73,56 @@ function EditorView(props) {
         right={rightComponent}
       />
       <section className="note-editor-container">
-        <Editor
-          disabled={collabProvider ? false : readOnly}
-          collabEdit={collabEdit}
-          onChange={onContentChange || nullCallback}
-          defaultValue={defaultValue}
-          {...editorConfig}
-          appearance="full-page"
-          placeholder={t('Notes.EditorView.main_placeholder')}
-          shouldFocus={!readOnly}
-          contentComponents={
-            <WithEditorActions
-              render={() => (
-                <>
-                  <aside ref={bannerRef} className={styles.banner}></aside>
-                  <MainTitle tag="h1" className={styles.title}>
-                    <Textarea
-                      ref={titleEl}
-                      rows="1"
-                      readOnly={!!readOnly}
-                      fullwidth={true}
-                      value={title}
-                      onChange={readOnly ? nullCallback : onTitleEvent}
-                      placeholder={defaultTitle}
-                      className={styles.titleInput}
+        <EditorContext>
+          <WithEditorActions
+            render={() => {
+              return (
+                <Editor
+                  disabled={collabProvider ? false : readOnly}
+                  collabEdit={collabEdit}
+                  onChange={onContentChange || nullCallback}
+                  defaultValue={defaultValue}
+                  legacyImageUploadProvider={imageUploadProvider(
+                    collabProvider
+                  )}
+                  {...editorConfig}
+                  appearance="full-page"
+                  placeholder={t('Notes.EditorView.main_placeholder')}
+                  shouldFocus={!readOnly}
+                  primaryToolbarComponents={[TestPlugin]}
```

Comment:
This seems to be a test, should it be removed before the merge?

Reply:
Updated